src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpCertificateHelper.cs

```diff
@@ -74,5 +75,50 @@ internal static class CertificateHelper
                 }
```

Comment:
GitHub isn't showing me the correct line of code.

Can you clarify your comment with the lines of code you're discussing?